packages/lerna-semantic-release-analyze-commits/package.json

```diff
@@ -5,12 +5,9 @@
     "url": "https://github.com/atlassian/lerna-semantic-release/issues"
   },
   "dependencies": {
+    "@semantic-release/commit-analyzer": "~2.0.0",
     "lerna-semantic-release-utils": "^4.0.0"
   },
-  "peerDependencies": {
-    "semantic-release": "semantic-release/semantic-release#a5cb9ea7f7a66657f95f0d913126e812adb6d9f8",
-    "@semantic-release/commit-analyzer": "~2.0.0"
```

Comment:
the analyzer was only used in this package, so doesn't need to be a peer. semantic-release itself was not used at all.